data/duke.txt

```diff
@@ -0,0 +1,4 @@
+T | 1 | borrow book
+D | 0 | return book | Sunday
+E | 0 | project meeting | Mon 2-4pm
+D | 1 | do homework | no idea :-p
```

Comment:
Consider adding this file to your .gitignore so that git doesn't track these changes.